src/Anon.ts

```diff
@@ -34,6 +34,7 @@ export default function anonymize(dict) {
     for(const key of Object.keys(dict)) {
         // Use default action or action specified in policy
         var rule = policy["default"];
+        //what happens if key is not in policy? ???
```

Comment:
`rule` is initially set to the default policy (the preceding line).

If there's a policy for that specific key, we use that instead.